impeller/entity/entity_unittests.cc

```diff
@@ -2812,6 +2812,31 @@ TEST_P(EntityTest, FailOnValidationError) {
       "");
 }
 
+TEST_P(EntityTest, CanRenderEmptyPathsWithoutCrashing) {
```

Comment:
Do you think we should follow through and do a full render?  We may have just punted the failure to later.

Reply:
Yeah I think that is a good idea, might as well make sure we don't crash somewhere else or hit a validation check.

Reply:
Bug found!